src/studio/src/designer/backend.Tests/Factories/ModelFactory/CustomXsd2JsonSchemaTests.cs

```diff
@@ -0,0 +1,29 @@
+using System.Xml;
+using Altinn.Studio.Designer.Factories.ModelFactory;
+using Designer.Tests.Utils;
+using Manatee.Json.Schema;
```

Comment:
Can we avoid Manatee in new and isolated code or do we need it? @rbendikaune knows more about the alternative and how far we are in replacing Manatee.

Reply:
This is a test showing how things work today, so Manatee is used with intention. If I'm using other frameworks there might be differences.

Reply:
Okay, good.